main.tf

```diff
@@ -26,8 +26,6 @@ resource "azurerm_role_assignment" "acr" {
 }
 
 resource "azurerm_user_assigned_identity" "aks" {
-  count = var.identity_ids != null ? 0 : 1
-
   location            = var.location
```

Comment:
You always want to create this identity even if the user providers an identity created outside of the module ?

Reply:
Do you need to keep the count with the condition based on:
```
(var.managed_identities.user_assigned_resource_ids) > 0
```

?

Reply:
i so no need to have this since the check is now in local.tf with this line

`    user_assigned = length(var.managed_identities.user_assigned_resource_ids) > 0 ? {
`